tests/test_date/good/rounding_option_fr.catala_fr

```diff
@@ -27,3 +27,8 @@ $ catala Interpret -s Test
 [RESULT] Computation successful! Results:
 [RESULT] r = vrai
 ```
+```catala-test-inline
+$ catala Interpret_Lcalc -s Test --avoid_exceptions --optimize
+[RESULT] Computation successful! Results:
+[RESULT] r = vrai
+```
```

Comment:
Please ensure to have newlines at the end of files. I think I fixed it, but I remember in particular that ``` wouldn't correctly parse as an end of test if not followed by a newline.